applications/api-cat/src/main/java/no/acat/harvester/ApiHarvester.java

```diff
@@ -44,11 +44,11 @@ public ApiHarvester(ElasticsearchService elasticsearchService, ApiDocumentBuilde
     public List<ApiDocument> harvestAll() {
         List<ApiDocument> result = new ArrayList<>();
 
-        List<ApiCatalogRecord> apiCatalog = getApiCatalog();
+        List<ApiRegistrationPublic> apiRegistrations = getApiRegistrations();
```

Comment:
Du endrer her fra ApiCatalogRecord tol ApiRegistrationPublic. Er dette bare en navneendring (name refactor) eller er det endret funksjonalitet også ?

Reply:
Det var den meningen av den hele kommitten - kanskje kommit melding var ikke klar nok

Så vi hadde en MVP av "registreringdatabase" i form av CSV. Nå erstatter vi det gradvis ut til ekte registreringdatabase. Det mener at det er rikgig, at søkeindeks are generert fra registreringmodell, og midlertidlig CSV løsning skulle bare følge den format, bare til å ha egle list-concatenation

Reply:
Bare for å få helt klarhet i dette: Den grunnleggende funksjonaliteten er at brukere skal kunne registerere egne apier, vi hadde ikke støtte for det før, og lastet bare inn noen pre-definerte fra en csv fil. Nå, derimot støtter vi en registreringsløsning for APIer og kan fase csv-filen gradvis ut. Korrekt?